frontend/mobilityai_react/App/Screens/Homescreen.js

```diff
@@ -24,21 +23,23 @@ export default class Homescreen extends Component {
             navigation: this.props.navigation
           })}>
             <View>
```

Comment:
If there's nothing in this view, you should remove it.

Reply:
Anything related to Patient Achievements was done by Rebecca, i don't want to remove it because i don't know if theres any other way to reach PatientAchievements

Reply:
To clarify, I meant to remove this view:

https://github.com/pgunasekara/mobilityai/blob/f952d93218b92f13116590c76b5f908419682f9c/frontend/mobilityai_react/App/Screens/Homescreen.js#L25-L27